docs/getting-started.md

```diff
@@ -1,76 +1,60 @@
-## Overview
+# Getting Started
 
-This document describes how to configure a forked Helm Broker bundles repository.
+This document describes how to configure your forked Helm Broker `bundles` repository.
 
 ## Prerequisites
 
-Before you start, install:
-
-* [Kyma](https://kyma-project.io/docs/latest/root/kyma#tutorials-tutorials) in version v0.4.1 or later
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in version v1.10.0 or later
-* [svcat CLI](https://svc-cat.io/docs/cli/) in version v1.10.0 or later
+* [Kyma](https://kyma-project.io/docs/latest/root/kyma#tutorials-tutorials) in version v0.4.1 or higher
```

Comment:
Shouldn't we link to the Kyma installation section instead of the tutorial?

Reply:
Good point, haven't noticed that, thanks! :)